docker/buildDocker.sh

```diff
@@ -26,11 +26,11 @@ parseCommandLineArgs()
 				"--jdk-boot-dir" | "-J")
 					bootDir="$1"; shift;;
 				"--openj9" | "-j9")
-					openJ9=true;;
+					buildVariant="openj9";;
 				"--use-eclipse-docker-files" | "-e" )
```

Comment:
I wonder if these vars should be more explicitly called `use-eclipse-openj9...`

Reply:
We could do, I just ran out of ideas to differentiate using the adopt openj9 dockerfiles, and the two sets that are in the Eclipse OpenJ9 repo.